apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/plugin/AbstractClassEnhancePluginDefine.java

```diff
@@ -104,6 +128,10 @@
         return new String[] {};
     }
 
+    protected List<WitnessMethod> witnessMethods() {
+        return Collections.emptyList();
```

Comment:
```suggestion
        return null;
```
Will be better performance.